### ISSUE TITLE
Module not Found

### ISSUE THREAD

**Issue by usysrc** [mercedes-benz/product-kit_vue#17]
Following the instructions from the README and using Vue 2.6.11 and Vuetify 2.6.6 I get the following error on `npm run serve`:

```
*[master][~/dev/js/my-app]$ npm run serve

> my-app@0.1.0 serve
> vue-cli-service serve

 INFO  Starting development server...


 ERROR  Failed to compile with 2 errors                                                                                                        1:26:39 PM

 error  in ./node_modules/@daimler/productkit-vue/src/theme/mbti/colors.js

Module not found: Error: Can't resolve '../../../../core/dist/web/styles/mbti/js/variables' in '/home/myuser/dev/js/my-app/node_modules/@daimler/productkit-vue/src/theme/mbti'

 error  in ./node_modules/@daimler/productkit-vue/src/theme/mbti/colors.js

Module not found: Error: Can't resolve '../../../../core/dist/web/styles/mbti/js/variables-dark' in '/home/myuser/dev/js/my-app/node_modules/@daimler/productkit-vue/src/theme/mbti'

ERROR in ./node_modules/@daimler/productkit-vue/src/theme/mbti/colors.js 3:0-77
Module not found: Error: Can't resolve '../../../../core/dist/web/styles/mbti/js/variables' in '/home/myuser/dev/js/my-app/node_modules/@daimler/productkit-vue/src/theme/mbti'
 @ ./src/plugins/vuetify.js 3:0-70 6:10-16
 @ ./src/main.js 3:0-40 6:2-9

ERROR in ./node_modules/@daimler/productkit-vue/src/theme/mbti/colors.js 4:0-86
Module not found: Error: Can't resolve '../../../../core/dist/web/styles/mbti/js/variables-dark' in '/home/myuser/dev/js/my-app/node_modules/@daimler/productkit-vue/src/theme/mbti'
 @ ./src/plugins/vuetify.js 3:0-70 6:10-16
 @ ./src/main.js 3:0-40 6:2-9

webpack compiled with 2 errors
```
Can you help with this problem?

**Comment by usysrc** [comment]
The paths in colors.js should be ../productkit-core/..
```
import * as tokens from "../../../../core/dist/web/styles/mbti/js/variables";
import * as tokensDark from "../../../../core/dist/web/styles/mbti/js/variables-dark";
```

**Comment by lukashar** [comment]
@usysrc Bug is fixed in release 2.0.1
